go.mod

```diff
@@ -62,8 +62,8 @@ require (
 	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
 	github.com/golang/geo v0.0.0-20190916061304-5b978397cfec
 	github.com/golang/protobuf v1.3.5
-	github.com/golang/snappy v0.0.1
-	github.com/google/go-cmp v0.5.2
+	github.com/golang/snappy v0.0.3
```

Comment:
are these deps updated intentionally? I'm just a bit nervous but we can always run this through Robot testing

Reply:
These forced upon me when upgrading sarama